src/components/Order/OrderItems.vue

```diff
@@ -1,32 +1,32 @@
 <template>
+  <div class="container">
+    <div class="row">
+      <div class="col-12 text-center">
+        <h4 class="pt-3">Order Id: - {{orderID}}</h4>
+      </div>
+    </div>
 
-    <div>
-
-        <h2>Products ordered</h2>
-
-        <div v-for="itr in this.lengthofOrderItems" :key="itr">
-
-            <div class="product-box">
-
-                <div class="col-4">
-                    <img class="img-fluid" v-bind:src="this.orderProducts[itr-1].imgUrl" alt="product-image">
-                </div>
-
-                <div class="col-8">
-
-                    <h3 class="product_name">{{this.orderProducts[itr-1].pName}}</h3>
-                    <h3 class="product_description">{{this.orderProducts[itr-1].pDescription}}</h3>
-                    <h3 class="product_price"><span>$</span>{{this.orderProducts[itr-1].pPrice}} per unit</h3>
-                    <h3 class="product_description">Quantity : {{this.orderProducts[itr-1].pQuantity}}</h3>
-                    <h3 class="product_description">Total Price : $ {{this.orderProducts[itr-1].pPrice*this.orderProducts[itr-1].pQuantity}}</h3>
-
-                </div>
-
-            </div>
-
+    <div v-for="itr in lengthofOrderItems" :key="itr" class="row mt-2 pt-3 justify-content-around">
+      <div class="col-2"></div>
+      <div class="col-md-3 embed-responsive embed-responsive-16by9">
+        <img v-bind:src="orderProducts[itr-1].imgUrl" class="w-100 card-img-top embed-responsive-item">
```

Comment:
iterate over item instead of index

Reply:
In each order, there can be multiple instances of the same product currently. Hence, there is no unique identity in case of product items. I can use Date.now() to create a unique string. Should I do that?